js/main.js

```diff
@@ -377,6 +366,7 @@ $(document).ready(function() {
     var legend = L.control({position: 'bottomright'});
     var dropDownCitySelection = $('#dropDownCitySelection');
     legend.onAdd = function () { return L.DomUtil.get('legend'); };
+    $("input[name=display]").change(updateLayers);
```

Comment:
Can you use single quotes? I think this seems to become our emerging coding style :)